Misc/NEWS.d/next/Core and Builtins/2021-02-18-15-12-30.bpo-42990.toAqBH.rst

```diff
@@ -0,0 +1,7 @@
+The :data:`types.FunctionType` constructor now inherits the current builtins if
+the *globals* parameter is used and the *globals* dictionary has no
```

Comment:
"if the *globals* parameter is used" seems redundant, since AFAICT that parameter is required and must be a dict.

Reply:
You're right: fixed.